### PR TITLE
translate documentation for database engines

### PR DESCRIPTION
**Pull request by qianmoQ**
I hereby agree to the terms of the CLA available at: https://yandex.ru/legal/cla/?lang=en

Changelog category (leave one):
- Documentation (changelog entry is not required)

Changelog entry (a user-readable short description of the changes that goes to CHANGELOG.md):
...

Detailed description / Documentation draft:
...

By adding documentation, you'll allow users to try your new feature immediately, not when someone else will have time to document it later. Documentation is necessary for all features that affect user experience in any way. You can add brief documentation draft above, or add documentation right into your patch as Markdown files in [docs](https://github.com/ClickHouse/ClickHouse/tree/master/docs) folder.

If you are doing this for the first time, it's recommended to read the lightweight [Contributing to ClickHouse Documentation](https://github.com/ClickHouse/ClickHouse/tree/master/docs/README.md) guide first.

Information about CI checks: https://clickhouse.tech/docs/en/development/continuous-integration/

### CODE REVIEW

File: docs/en/engines/table-engines/integrations/kafka.md [hunk 1]
```diff
@@ -48,7 +48,7 @@ Optional parameters:
 -   `kafka_row_delimiter` — Delimiter character, which ends the message.
 -   `kafka_schema` — Parameter that must be used if the format requires a schema definition. For example, [Cap’n Proto](https://capnproto.org/) requires the path to the schema file and the name of the root `schema.capnp:Message` object.
 -   `kafka_num_consumers` — The number of consumers per table. Default: `1`. Specify more consumers if the throughput of one consumer is insufficient. The total number of consumers should not exceed the number of partitions in the topic, since only one consumer can be assigned per partition.
--   `kafka_max_block_size` — The maximum batch size (in messages) for poll (default: `max_block_size`).
+-   `kafka_max_block_size` - The maximum batch size (in messages) for poll (default: `max_block_size`, If `kafka_max_block_size` parameter setting, will use `max_block_size` and `kafka_max_block_size` Minimum value in size.).
```
Comment (qoega) [inline docs/en/engines/table-engines/integrations/kafka.md:51]:
```suggestion
-   `kafka_max_block_size` - The maximum batch size (in messages) for poll (default: `max_block_size`, but not larger than current `max_block_size`.
```

File: docs/en/whats-new/changelog/2017.md [hunk 2]
```diff
@@ -1,266 +1,267 @@
----
+docs/en/whats-new/changelog/2017.md---
 toc_priority: 79
 toc_title: '2017'
 ---
 
 ### ClickHouse Release 1.1.54327, 2017-12-21 {#clickhouse-release-1-1-54327-2017-12-21}
 
-This release contains bug fixes for the previous release 1.1.54318:
+此版本包含`1.1.54318`版本的错误修复:
```
Comment (qoega) [inline docs/en/whats-new/changelog/2017.md:8]: docs/en/* should not be translated

**Review by qoega — CHANGES_REQUESTED** [review]
Probably some files were added by accident?

File: docs/en/engines/table-engines/integrations/kafka.md (hunk 1 above)
Comment (qianmoQ) [reply docs/en/engines/table-engines/integrations/kafka.md:51]: > Probably some files were added by accident?

Sorry, I'll revise it

**Review by qoega — APPROVED** [review]

**Pull request closed without merging**